facebook/react-native
# Remove global babelHelpers and regenerator

Follow up to https://github.com/facebook/metro/commit/8932a9caa8f9ffe82160f2da25342a0bde3bbb26

Babel helpers and regenerator runtime will be imported automatically from `@babel/runtime`. We still need to add the global regeneratorRuntime for jest tests since we disable babel-runtime currently.

Test Plan:
----------
- Run RN tester and make sure it works fine for screens using async functions.
- Inspect the generated bundle to make sure helpers and regenerator are imported from @babel/runtime
- Made sure the global babelHelpers and the old regenerator-runtime are not in the generated bundle.
- Run jest tests

Release Notes:
--------------
Help reviewers and the release process by writing your own release notes. See below for an example.

[GENERAL] [ENHANCEMENT] [polyfills] - Remove global babelHelpers and regenerator
